controlplane/src/core/env.schema.ts

```diff
@@ -39,4 +39,10 @@ export const envVariables = z.object({
   S3_STORAGE_URL: z.string(),
   SMTP_USERNAME: z.string().optional(),
   SMTP_PASSWORD: z.string().optional(),
+  /**
+   * Billing
+   */
+  STRIPE_SECRET_KEY: z.string().optional(),
```

Comment:
If someone sets `webhookSecret` without `secret` is it catched? Can we have constraints?

Reply:
I dont think we need to do that much handholding, but I added a check in the schema